### DISCUSSION TITLE
Deprecate support for CycloneDX schema version 1.0

### DISCUSSION THREAD

**Discussion by madpah** [CycloneDX/cyclonedx-python-lib#217] [Polls]
As called out in #151 by @jkowalleck, `1.0` is old and very sparse compared to the more recent versions of the CycloneDX schema.

This ticket aims to collect any feedback from the community as to whether:
- This library is used to product BOMs in schema version `1.0`
- Anyone has any strong feelings / requirements for us to keep supporting `1.0`

**Comment by jkowalleck** [comment]
yes, this "feature" would cause breaking changes, as it cuts of an existing feature, and might remove existing classes/structures.

benefit: removing 1.0 capabilities removed complexity from the normalizers as no code-branching for is required, for features that exist from spec 1.1 on.
